### PR TITLE
Changed default capsule axis to vertical

### LINKED ISSUE #5813: Have Capsule Collider Vertical By Default
**Operating system or device - Godot version:**
Any OS, any Godot version

**Issue description** (what happened, and what was expected):
As it is now when testing I have to rotate the capsule collider 90 degrees on the X axis before I can use it. At first I thought it was just weird and I was using it wrong (keeping it horizontal), but that makes collisions act strangely. The capsule collider icon in the inspector shows an upright, vertical capsule so it's confusing.

**Steps to reproduce:**

**Link to minimal example project** (optional but very welcome):

### PR DESCRIPTION
**Pull request by Chaosus**
Changed `CapsuleMesh`, `CapsuleShape` and it's gizmos to vertical.

Closes https://github.com/godotengine/godot/issues/5813

### CODE REVIEW

File: modules/bullet/shape_bullet.h [hunk 1]
```diff
@@ -48,6 +49,17 @@ class ShapeBullet;
 class btCollisionShape;
 class ShapeOwnerBullet;
 class btBvhTriangleMeshShape;
+class btCapsuleShape;
+
+class CapsuleShapeY : public btCapsuleShape {
```
Comment (AndreaCatania) [inline modules/bullet/shape_bullet.h]: Why don't use `btCapsuleShape` directly? We don't have multiple capsules oriented differently after all, so why be so specific?

**Review by AndreaCatania — COMMENTED** [review]
This looks good to me, I've just a little comment that you can find below, but other than that this is a good change!

File: modules/bullet/shape_bullet.h (hunk 1 above)
Comment (Chaosus) [reply modules/bullet/shape_bullet.h]: Ah, ok, I will fix that if it's not desired.

Comment (Chaosus) [reply modules/bullet/shape_bullet.h]: Done!

**Comment by Calinou** [comment]
I was working on doing this :slightly_frowning_face: 

Anyway, your PR seems fine to me.

**Comment by Chaosus** [comment]
Sometimes thoughts are crossed 😄 sorry about that )

**Review by AndreaCatania — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**